src/commands/BootstrapCommand.js

```diff
@@ -204,8 +204,8 @@ export default class BootstrapCommand extends Command {
         .forEach((dependency) => {
           // get Package of dependency
           const dependencyPackage = this.packageGraph.get(dependency).package;
-          // get path to dependency and its package.json
-          const dependencyLocation = dependencyPackage.location;
+          // get path to dependency and its scope
+          const { location: dependencyLocation, scope: dependencyScope } = dependencyPackage;
```

Comment:
But the scope is part of the name, as returned by [`package.name`](https://github.com/lerna/lerna/blob/master/src/Package.js#L10-L12), right?

It's just returning the `name` value from the package.json, which has the scope in it.